.github/workflows/release.yml

```diff
@@ -1,61 +1,84 @@
-# Modified Release github action. Replace the current with this when ready.
+# This is the workflow for building release artifacts.
+# Optionally, creating a release with those artifacts is possible by pushing a tag.
 ---
-name: Release New
+name: Release
```

Comment:
Making this change (putting it back, really) causes the Actions tab to show multiple "Release" workflows:

<img width="1312" alt="Screen Shot 2022-03-11 at 1 01 03 PM" src="https://user-images.githubusercontent.com/18729796/157934721-f15751cc-c731-4424-850f-820856a4d0db.png">

The correct one can be determined by hovering over the link to view the file name (should be `release.yml`). I just disabled the unused workflows...but I'm wondering if there is a way to remove them permanently. Are they needed for something...or some amount of time/retention?

Reply:
I did not explicitly see language on deleting a Workflow in the GH docs. My guess is they are meant to remain for historical purposes. If we really want to remove them, we can try deleting all of the Workflow Runs in each Workflow and see if that does it, but that might not be necessary as they are now disabled anyways.